workflows/magmap.nf

```diff
@@ -281,17 +281,17 @@ workflow MAGMAP {
     GUNZIP_CONTIGS(ch_no_gff.gzipped)
     ch_versions = ch_versions.mix(GUNZIP_CONTIGS.out.versions)
 
+    // PROKKA on the genomes that lack gff
     PROKKA(GUNZIP_CONTIGS.out.file.mix(ch_no_gff.unzipped), [], [])
     ch_versions = ch_versions.mix(PROKKA.out.versions)
 
-    // PROKKA on the genomes that lack gff
+    // Mix genome entries that were not sent to Prokka with those that were
     ch_finished_genomes = ch_genomes
```

Comment:
Maybe a better name for this channel?
something like `ch_collected_genomes` or similar since `finished` might be too generic. but I am being too picky maybe.